src/libpanic_unwind/seh.rs

```diff
@@ -215,22 +213,17 @@ extern "C" {
     static TYPE_INFO_VTABLE: *const u8;
 }
 
-// We use #[lang = "msvc_try_filter"] here as this is the type descriptor which
+// We use #[lang = "eh_catch_typeinfo"] here as this is the type descriptor which
 // we'll use in LLVM's `catchpad` instruction which ends up also being passed as
 // an argument to the C++ personality function.
 //
 // Again, I'm not entirely sure what this is describing, it just seems to work.
-#[cfg_attr(not(test), lang = "msvc_try_filter")]
-static mut TYPE_DESCRIPTOR1: _TypeDescriptor = _TypeDescriptor {
+#[cfg_attr(bootstrap(msvc_try_filter))]
```

Comment:
```suggestion
#[cfg_attr(bootstrap, msvc_try_filter)]
```

Reply:
```suggestion
#[cfg_attr(bootstrap, lang = "msvc_try_filter")]
```

To fix https://github.com/rust-lang/rust/pull/65646#issuecomment-549032221

Reply:
Thanks. Note to self: allow time to wake up properly before fixing code.